docs/thousand-validators.md

```diff
@@ -0,0 +1,61 @@
+---
+id: thousand-validators
+title: Thousand Validators Programme
+sidebar_label: Thousand Validators Programme
+---
+
+The Thousand Validators Programme is an initiative by Web3 Foundation and Parity Technologies to use
+the funds held by both organizations to nominate validators in the community.
+
+It serves two major purposes: 1) to give validators a structured on-ramp to join the active set of
+validators on Kusama and Polkadot and 2) to further decentralize the validator active set.
+
+## How it Works
+
+The nominating backend will routinely change its nominations at every era (or every 4 eras on
+Kusama). It will nominate validators which fit all the requirements and are eligbile. Of this pool,
+it will nominate as many as possible although some validators which are eligible might not receive
+nominations every round (due to the constraint of nominating a maximum of 16 validators per
+nominator). If a validator performs well during a single nomination period (the time after a new
```

Comment:
hmm..interesting.. my node keeps dropping for the past few days..;). could you explain a little bit more on "performs well"?